src/Pact/Compile.hs

```diff
@@ -481,25 +483,30 @@ interface = do
       ihash = ModuleHash . pactHash . encodeUtf8 . _unCode $ code
   (bd,ModuleState{..}) <- withModuleState (initModuleState iname ihash) $
             bodyForm $ specialForm $ \r -> case r of
-              RDefun -> return emptyDef
+              RDefun -> return $ defSig Defun
               RDefconst -> return defconst
               RUse -> return useForm
+              RDefpact -> return $ defSig Defpact
```

Comment:
adding defpacts currently makes no sense. I think we could do this in the future if we could specify step with yield + resume signatures, but otherwise, it gives the user 0 information about what the implementation is or should be.

Reply:
It's a baby step though, that can be further enumerated later. By including, it allows `fungible` to express `move`, and to a certain extent, the arguments to a pact indicate what the intent is. The user doesn't have to know anything about the cont as it will be uniform for any cross chain exec, as the continuation is spit out by the execution, and the proof comes from the endpoint irrespective of the contents. Indeed this might be a benefit for tokens that have more information attached.

I think to a certain extent we've oversold interfaces as a mechanism for safety, and undersold them as a mechanism for interop. As it is, FV for interfaces is still minimal, while the interop benefits are proven. Adding defpact helps interop.